### PR TITLE
Tentatively disable some of the web-api tests (#1334)

### PR DESCRIPTION
**Pull request by seratch**
###  Summary

This pull request is a kind of workaround for #1334 -- with this change, we can make a new release of web-api package shortly.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/web-api/src/WebClient.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (seratch) [inline packages/web-api/src/WebClient.ts:1]: These disabling comments are not required for passing tests, though.

File: packages/web-api/package.json [hunk 2]
```diff
@@ -49,21 +49,21 @@
     "@slack/types": "^2.0.0",
     "@types/is-stream": "^1.1.0",
     "@types/node": ">=12.0.0",
-    "axios": "0.21.1",
+    "axios": "^0.21.4",
```
Comment (seratch) [inline packages/web-api/package.json:52]: We can upgrade axios in this PR

**Review by filmaj — APPROVED** [review]
LGTM.

Just for the record: these changes are to address the vulnerability in older versions of axios. The tradeoff for reducing test coverage is so that we can move forward with less vulnerabilities in dependencies.

**Comment by srajiang** [comment]
Thank you @filmaj and @seratch for following up on this - I don't have much to add, but here's a comment to let you know I'm following along on #1334 and related investigation.

**Comment by filmaj** [comment]
FYI, I believe we should close this PR in favour of #1369

**Comment by seratch** [comment]
@filmaj Thanks! We can close this one

**Pull request closed without merging**